src/handlers/dns/DnsStreamHandler.cpp

```diff
@@ -20,6 +20,8 @@
 
 namespace visor::handler::dns {
 
+thread_local static DnsCacheData cached_dns_layer;
```

Comment:
@weyrick  thread_local cannot be a member variable:
https://stackoverflow.com/a/10999179